patches/chrome-browser-ui-views-frame-browser_view.cc.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
```

Comment:
I think this patch was added by mistake?

Reply:
Removed.

Reply:
OMG - sorry about this 😓